src/test/java/com/wikia/webdriver/common/core/helpers/Emulator.java

```diff
@@ -15,6 +15,15 @@
           .put("mobile", false)
           .build(),
       "Mozilla/5.0 (Linux; Android 6.0; Nexus 5 Build/MRA58N) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/66.0.3359.170 Mobile Safari/537.36"),
+  GOOGLE_NEXUS_5_DEFAULT( // FIXME: this is workaround for mobile: false flag, remove it if original flag will be reverted
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Take the required action to fix the issue indicated by this comment. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](http://sonar-s4/coding_rules#rule_key=squid%3AS1134)
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Move this trailing comment on the previous empty line. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](http://sonar-s4/coding_rules#rule_key=squid%3ATrailingCommentCheck)

Reply:
How about removing it and re-using https://github.com/Wikia/selenium-tests/blob/0c4a4ddf18617f19b87abc1a265b7bea43f8777d/src/test/java/com/wikia/webdriver/common/core/helpers/Emulator.java#L47 instead?